sequence_task_1/student/include/impl.h

```diff
@@ -8,33 +8,77 @@ class StaticPlaylist
 {
 public:
     /** @todo Member traits */
+    using value_type = typename Container::value_type;
+    using size_type = typename Container::size_type;
+    using difference_type = typename Container::difference_type;
+    using reference = typename Container::reference;
+    using const_reference = typename Container::const_reference;
+    using iterator = typename Container::iterator;
+    using const_iterator = typename Container::const_iterator;
+    using pointer = typename Container::pointer;
 
     /** @todo Iterators */
+    const_iterator begin() const {
+        return m_tracklist.cbegin();
+    }
 
-    StaticPlaylist() = default;
+    const_iterator end() const {
+        return m_tracklist.cend();
+    }
+
+    StaticPlaylist() {
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c80-use-default-if-you-have-to-be-explicit-about-using-the-default-semantics